mesheryctl/pkg/utils/scripts.go

```diff
@@ -141,20 +145,16 @@ func GenerateConfigAKS() error {
 	}
 	
 	fetch_aks_script() {
-		echo -n "enter your resource group"
-		printf "\n"
-		read -r resource_group
-		echo -n "enter your aks cluster name"
 		printf "\n"
-		read -r aks_name
-		az aks get-credentials --resource-group "${resource_group}" --name "${aks_name}" --file "${TARGET_FILE}"
+		az aks get-credentials --resource-group "${Resource_Group}" --name "${Cluster_Name}" --file "${TARGET_FILE}"
```

Comment:
Where are we checking for the `az` dependency?

Reply:
We are not as it is a prereq, or do we want to make script fetch the az too if not found?

Reply:
Great question. For now, if we just gracefully handle the potentially missing prerequisite error, we can merge this PR and put out a nice-to-have PR for a contributor to consider picking up two auto install `az`￼￼￼.

Reply:
Done!

Reply:
I wonder if this is the same formatting issue being run into by Prateek T.?

Reply:
Yes, the formatting issue is due to the presence of editor config for golang files ig. Every IDE formats according to golang standards but after seeing editorconfig the presets are overridden resulting in failure of gofmt. 😓

Reply:
Got it, ok. Prateek's PR was held up on account of this perhaps for longer than it should have been in retrospect.